internal/supervisor/server/server.go

```diff
@@ -526,6 +531,10 @@ func runSupervisor(ctx context.Context, podInfo *downward.PodInfo, cfg *supervis
 		}
 
 		c := ptls.Default(nil)
+		// Remove "h2" from the list for now, until we have a better idea of how to mitigate
+		// potential http2 rapid reset vulnerabilities. This disables serving requests using http2.
```

Comment:
Just wondering if this (only http 1.1) opens us up to other issues. 🤔 
I'm not sure if this means we are making a trade that we might be unhappy with?

Reply:
This had the unintended consequence of breaking a pre-existing GKE Ingress upon upgrade of the Supervisor. That Ingress had a backend service, and that Service was configured with the `"[cloud.google.com/app-protocols](http://cloud.google.com/app-protocols)": "{\"https\":\"HTTP2\"}"` annotation, so the Ingress was not able to route traffic anymore after upgrading the Supervisor to a version which does not accept http2 traffic.

I think we should roll back this part of the PR for now. I'll create a new PR for it.